Cargo.toml

```diff
@@ -16,7 +16,7 @@ feature_gtk3 = ["gdk3x11", "gdk3wayland", "gtk3"]
 feature_gtk4 = ["gdk4x11", "gdk4wayland", "gtk4"]
 feature_pipewire = ["pw", "libc"]
 log = ["tracing"]
-tokio_runtime = ["zbus/tokio", "tokio"]
+tokio = ["zbus/tokio", "dep:tokio"]
```

Comment:
You also need to do changes in code where you were using feature = "tokio_runtime". 

If you are really motivated to do that, please drop the log feature -> make it use tracing as it is an optional dependency already. Rename feature_gtk4/feature_gtk3/feature_pipewire to gtk4/gtk3/pipewire

Reply:
Done, but I wonder if it is a good idea to rename the others.... well it is in its own commit which can be removed.